src/utils/appInstaller/index.ts

```diff
@@ -167,7 +171,7 @@ async function _installExternalApp(
     const shortName = name.split('.')[0]
     const initialVersionArray = toApmVersionArray('1.0.0')
     const contentUriHttpFromPublicGateway = utf8ToHex(
-      `http:${ipfsGateway}${getContentHash(contentURI)}`
+      'http:' + joinIpfsLocation(ipfsGateway, getContentHash(contentURI))
```

Comment:
@0xGabi This was the actual line creating the bug, where the URL was not properly joined so the second vault instance was trying to be fetched from a broken url